lib/foreman_inventory_upload/async/upload_report_job.rb

```diff
@@ -21,7 +21,7 @@ def perform(filename, organization_id)
       end
 
       def command
-        File.join(File.dirname(@filename), ForemanInventoryUpload.upload_script_file)
+        ['/usr/bin/sh', File.join(File.dirname(@filename), ForemanInventoryUpload.upload_script_file)]
```

Comment:
Not sure if it's relevant, but on Debian:
```console
$ ls /usr/bin/sh
ls: cannot access '/usr/bin/sh': No such file or directory
```

Traditionally this was always `/bin/sh`.

Reply:
This plugin depends on Katello, so not too relevant. But /bin/sh is indeed clearer